Mathlib/GroupTheory/Subgroup/MulOpposite.lean

```diff
@@ -20,44 +21,61 @@ variable {G : Type*} [Group G]
 
 namespace Subgroup
 
+/-- pull a subgroup back to an opposite subgroup along `unop`-/
```

Comment:
```suggestion
/-- Pull a subgroup back to an opposite subgroup along `MulOpposite.unop`. -/
```
and similar capitalization / qualification / `.` on the other 7 versions of this docstring